javascript/javascript1/week3/phone-usage.js

```diff
@@ -0,0 +1,38 @@
+const activities = [];
+let date = new Date();
+let today = date.toLocaleDateString();
+
+function addActivity(date, activity, period) {
+    activities.push({ date, activity, period });
+}
+// adding acctivities to the empty array
+addActivity(today, "YouTube", 30);
+addActivity(today, "Facebook", 34);
+addActivity(today, "Twitter", 14);
+//addActivity(today, "Instagram", 12); // no smartphoning alert pops up when it reachs the limit
+
+console.log(activities);
+
+function showStatus(data) {
```

Comment:
I recommend you to rename `data` to `activities` and check this array length. if it is empty log that and return
```js
    if (activities.length === 0) {
      console.log('Add some activities before calling showStatus');
      return;
    }
```